bucket_58/python-vdirsyncer/specification

```diff
@@ -0,0 +1,55 @@
+DEF[PORTVERSION]=	0.16.7
+# ------------------------------------------------------------------------
+
+NAMEBASE=		python-vdirsyncer
+VERSION=		${PORTVERSION}
+KEYWORDS=		python devel
+VARIANTS=		py27 py36 py37
+SDESC[py27]=		Synchronize calendars and contacts (PY 27)
+SDESC[py36]=		Synchronize calendars and contacts (PY 36)
+SDESC[py37]=		Synchronize calendars and contacts (PY 37)
+HOMEPAGE=		https://vdirsyncer.pimutils.org
+CONTACT=		Alexandre_C_Guimarães[privacychain.ch@privacychain.ch]
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		PYPI/v/vdirsyncer
+DISTFILE[1]=		vdirsyncer-0.16.7.tar.gz:main
+
+SPKGS[py27]=		single
+SPKGS[py36]=		single
+SPKGS[py37]=		single
+
+OPTIONS_AVAILABLE=	GOOGLE PY27 PY36 PY37
+OPTIONS_STANDARD=	none
+VOPTS[py27]=		PY27=ON PY36=OFF PY37=OFF
+VOPTS[py36]=		PY27=OFF PY36=ON PY37=OFF
+VOPTS[py37]=		PY27=OFF PY36=OFF PY37=ON
+
+LICENSE=		BSD3CLAUSE:single
+LICENSE_FILE=		BSD3CLAUSE:{{WRKSRC}}/LICENSE
+LICENSE_SCHEME=		solo
+
+FPC_EQUIVALENT=		deskutils/py-vdirsyncer
+
+[GOOGLE].RUN_DEPENDS_ON=		python-requests-oauthlib:single:?
```

Comment:
I have two doubts. It support py27-p36 but not py37 yet.

How make it explicit? Also, should the option be set separated for each VOPTS -> [PY27].[GOOGLE].RUN_DEPENDS_ON?

There is a second possible option to support [EteSync](https://www.etesync.com/) but already removed on master. I do not think it worth to add.

Thanks!